make.js

```diff
@@ -458,7 +481,7 @@ CLI.test = function(/** @type {{ suite: string; node: string; task: string }} */
     function runTaskTests(taskName) {
         banner('Testing: ' + taskName);
         // find the tests
-        var nodeVersions = argv.node ? new Array(argv.node) : getTaskNodeVersion(buildTasksPath, taskName);
+        var nodeVersions = argv.node ? new Array(argv.node) : [Math.max(...getTaskNodeVersion(buildTasksPath, taskName))];
```

Comment:
not blocking, maybe for another PR, but it would be good to run the tests under each node version supported by the task

Reply:
Before we made this change, it will test `HelmDeployV0_Node20 `for each node version, running Mocha Suits for node 10, 16 and 20 versions extracted from task.json execution. I think we only need to pick the highest node version from it, use node 20 to run `HelmDeployV0_Node20 `and no other versions needed.

```
"execution": {
    "Node10": {
      "target": "src//helm.js"
    },
  "Node16": {
        "target": "src//helm.js"
    },
  "Node20": {
        "target": "src//helm.js"
    }
  },
```